src/main/java/com/eppo/sdk/deserializer/BanditsDeserializer.java

```diff
@@ -53,9 +53,10 @@ public Map<String, BanditParameters> deserialize(
             JsonNode coefficientsNode = modelDataNode.get("coefficients");
 
             Map<String, BanditCoefficients> coefficients = new HashMap<>();
-            coefficientsNode.iterator().forEachRemaining(actionCoefficientsNode -> {
-                BanditCoefficients actionCoefficients = this.parseActionCoefficientsNode(actionCoefficientsNode);
-                coefficients.put(actionCoefficients.getActionKey(), actionCoefficients);
+            Iterator<Map.Entry<String, JsonNode>> coefficientIterator = coefficientsNode.fields();
+            coefficientIterator.forEachRemaining(field -> {
+                BanditCoefficients actionCoefficients = this.parseActionCoefficientsNode(field.getValue());
+                coefficients.put(field.getKey(), actionCoefficients);
```

Comment:
while we're leaving `actionKey` in the action coefficients for now, that is redundant and may be later removed, so I don't want to count on it being there and instead use the Object key as the action key.